linkis-computation-governance/linkis-client/linkis-computation-client/src/test/java/org/apache/linkis/computation/client/InteractiveJobTest.java

```diff
@@ -29,7 +29,7 @@ public static void main(String[] args) {
     SubmittableInteractiveJob job =
         LinkisJobClient.interactive()
             .builder()
-            .setEngineType("hive-2.3.3")
+            .setEngineType("hive-3.1.3")
```

Comment:
Should use org.apache.linkis.manager.label.conf.LabelCommonConfig#HIVE_ENGINE_VERSION